content/library/advanced-features/configuration.md

```diff
@@ -70,6 +70,7 @@ Streamlit provides four different ways to set configuration options:
 
    ```bash
    export STREAMLIT_SERVER_PORT=80
+   export STREAMLIT_SERVER_COOKIE_SECRET=dontforgottochangeme
```

Comment:
What does this addition add to the documentation?

Reply:
There are some multi-word configuration options like `server.cookieSecret` for which I had some trouble figuring out that these needs to be passed as "capitalized snake case" environment variables. E.g. `server.cookieSecret` -> `STREAMLIT_SERVER_COOKIE_SECRET` (and not `STREAMLIT_SERVER_COOKIESECRET`). As it took me some time to figure this out I thought it would be usefull to have an example of these multi-word configuration options is in the docs.

Reply:
Hey @randyzwitch was my clarification enough to get this PR accepted? Please let me know if there's anything I can do to improve the PR.